src/components/basket-item/basket-item.jsx

```diff
@@ -4,27 +4,69 @@ var React = require('react');
 module.exports = React.createClass({
   propTypes: {
     currencySymbol: React.PropTypes.string,
+    freeText: React.PropTypes.string,
+    removeText: React.PropTypes.string,
     price: React.PropTypes.number,
+    title: React.PropTypes.node,
+    toggleDescription: React.PropTypes.bool,
+    handleRemove: React.PropTypes.func,
     children: React.PropTypes.node
   },
+  getInitialState: function() {
+    return {
+      descriptionVisibility: !this.props.toggleDescription
+    };
+  },
   getDefaultProps: function() {
     return {
-      currencySymbol: '£'
+      title: null,
+      price: null,
+      handleRemove: null,
+      currencySymbol: '£',
```

Comment:
Should this component use react-intl rather than having the caller provide a symbol and value for prices?